test/ios/DidomiUITests/DidomiUITests.swift

```diff
@@ -120,7 +121,7 @@ class DidomiUITests: XCTestCase {
     tapButton(in: app, name: "updateSelectedLanguage")
     assertResult(in: app, name: "updateSelectedLanguage", expected: "updateSelectedLanguage-OK")
 
-    testLastEvent(app: app, name:"on_language_updated\"fr\"")
+    testLastEvent(app: app, name:"on_language_updated\n  \"fr\"")
```

Comment:
why this change? Could trimming the event description could solve any future issues like this?